### PR TITLE
Fix golint error

### PR DESCRIPTION
**Pull request by gongweibao**
Fix https://github.com/PaddlePaddle/cloud/issues/609

### CODE REVIEW

File: go/pkg/apis/paddlepaddle/v1alpha1/types.go [hunk 1]
```diff
@@ -92,20 +101,28 @@ type TrainerJobScaleStatus struct {
...
 type ResourceState string
 
 const (
-	ResourceStateNone      ResourceState = ""
-	ResourceStateStarting                = "starting"
-	ResourceStateRunning                 = "running"
-	ResourceStateFailed                  = "failed"
-	ResourceStateSucceeded               = "succeeded"
+	// ResourceStateNone is the none name of ResourceState.
+	ResourceStateNone ResourceState = ""
```
Comment (m3ngyang) [inline go/pkg/apis/paddlepaddle/v1alpha1/types.go]: ResourceStateNone is the initial state of training job

File: go/pkg/apis/paddlepaddle/v1alpha1/types.go [hunk 2]
```diff
@@ -92,20 +101,28 @@ type TrainerJobScaleStatus struct {
...
+	// ResourceStateStarting is the starting name of ResourceState.
+	ResourceStateStarting = "starting"
```
Comment (m3ngyang) [inline go/pkg/apis/paddlepaddle/v1alpha1/types.go]: ResourceStateStarting is the starting state of training job

File: go/pkg/apis/paddlepaddle/v1alpha1/types.go [hunk 3]
```diff
@@ -92,20 +101,28 @@ type TrainerJobScaleStatus struct {
...
+	// ResourceStateRunning is the  running name of ResourceState.
+	ResourceStateRunning = "running"
```
Comment (m3ngyang) [inline go/pkg/apis/paddlepaddle/v1alpha1/types.go]: ResourceStateStarting is the running state of training job

File: go/pkg/apis/paddlepaddle/v1alpha1/types.go [hunk 4]
```diff
@@ -92,20 +101,28 @@ type TrainerJobScaleStatus struct {
...
+	// ResourceStateFailed is the failed name of ResourceState.
+	ResourceStateFailed = "failed"
```
Comment (m3ngyang) [inline go/pkg/apis/paddlepaddle/v1alpha1/types.go]: ResourceStateFailed is the failed state of training job

File: go/pkg/apis/paddlepaddle/v1alpha1/types.go [hunk 5]
```diff
@@ -92,20 +101,28 @@ type TrainerJobScaleStatus struct {
...
+	// ResourceStateSucceeded is the succeeded name of ResourceState
+	ResourceStateSucceeded = "succeeded"
```
Comment (m3ngyang) [inline go/pkg/apis/paddlepaddle/v1alpha1/types.go]: ResourceStateSucceeded is the succeeded state of training job

File: go/pkg/apis/paddlepaddle/v1alpha1/types.go (hunk 1 above)
Comment (gongweibao) [reply go/pkg/apis/paddlepaddle/v1alpha1/types.go]: Done.

File: go/pkg/apis/paddlepaddle/v1alpha1/types.go (hunk 2 above)
Comment (gongweibao) [reply go/pkg/apis/paddlepaddle/v1alpha1/types.go]: Done.

File: go/pkg/apis/paddlepaddle/v1alpha1/types.go (hunk 4 above)
Comment (gongweibao) [reply go/pkg/apis/paddlepaddle/v1alpha1/types.go]: Done.

File: go/pkg/apis/paddlepaddle/v1alpha1/types.go (hunk 5 above)
Comment (gongweibao) [reply go/pkg/apis/paddlepaddle/v1alpha1/types.go]: Done.

**Comment by gongweibao** [comment]
@typhoonzero create an ISSUE and fix it yet.

**Pull request merged**